### PR TITLE
Add regex for syslog timestamp in extract_log

### PR DESCRIPTION
**Pull request by kellyyeh**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205

#### What is the motivation for this PR?
Warm-reboot test is failing at extracting timestamp from syslog

#### How did you do it?
Add regex for this timestamp format: 2023-02-28T15:42:27.355561+00:00

#### How did you verify/test it?
Warm reboot test originally failed at preboot when parsing through syslog. Adding the regex fixed the failure.
Ran convert_date with all three formats of syslog timestamp logs

### CODE REVIEW

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Passed<br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>ansible/library/extract_log.py:76:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:77:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:78:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:79:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:80:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:81:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:82:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:83:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:84:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:85:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:86:1: F403 'from ansible.module_utils.basic import *' used; unable to detect undefined names<br/>...<br/>[truncated extra lines, please run pre-commit locally to view full check results]<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

File: ansible/library/extract_log.py [hunk 1]
```diff
@@ -116,7 +116,6 @@ def extract_number(s):
     else:
         return int(ns[0])
 
-
```
Comment (vaibhavhd) [inline ansible/library/extract_log.py:119]: Minor: two places where line breaks are removed. Double line break is a pip8 suggested format.

**Review by vaibhavhd — APPROVED** [review]
LGTM. Please mention how this change was tested.

**Comment by wangxin** [comment]
The [pre-commit](http://pre-commit.com/) check detected issues in the files touched by this pull request.
The detected issues may be old or new. For new issues, please try to fix them.

For old issues, it is not mandatory to fix them because they were not caused by this change. It is unfair to blame
author of this pull request. But if you can take extra effort to fix the old issues as well, that would be great!

Detailed pre-commit check results:
<samp>trim trailing whitespace.................................................Passed<br/>fix end of files.........................................................Passed<br/>check yaml...........................................(no files to check)Skipped<br/>check for added large files..............................................Passed<br/>check python ast.........................................................Passed<br/>flake8...................................................................Failed<br/>- hook id: flake8<br/>- exit code: 1<br/><br/>ansible/library/extract_log.py:76:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:77:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:78:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:79:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:80:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:81:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:82:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:83:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:84:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:85:1: E402 module level import not at top of file<br/>ansible/library/extract_log.py:86:1: F403 'from ansible.module_utils.basic import *' used; unable to detect undefined names<br/>...<br/>[truncated extra lines, please run pre-commit locally to view full check results]<br/></samp>

To run the pre-commit checks locally, you can follow below steps:
1. Ensure that default python is python3. In sonic-mgmt docker container, default python is python2. You can run
   the check by activating the python3 virtual environment in sonic-mgmt docker container or outside of sonic-mgmt
   docker container.
2. Ensure that the `pre-commit` package is installed:
```
sudo pip install pre-commit
```
3. Go to repository root folder
4. Install the pre-commit hooks:
```
pre-commit install
```
5. Use pre-commit to check staged file:
```
pre-commit
```
6. Alternatively, you can check committed files using:
```
pre-commit run --from-ref <commit_id> --to-ref <commit_id>
```

File: ansible/library/extract_log.py (hunk 1 above)
Comment (kellyyeh) [reply ansible/library/extract_log.py:119]: Updated

**Comment by kellyyeh** [comment]
> LGTM. Please mention how this change was tested.

Updated testing

**Review by vaibhavhd — APPROVED** [review]

**Pull request merged**